classes/form/FormField.php

```diff
@@ -211,4 +220,13 @@ public function getAutocompleteAttribute(): string
     {
         return $this->autocomplete;
     }
+
+        $this->attr = $atts;
```

Comment:
You forgot some part of your code. The function header is missing.

Reply:
Hello @nicosomb it's fixed